build_tools/stable_diffusion_testing.py

```diff
@@ -64,6 +65,75 @@ def test_loop(device="vulkan", beta=False, extra_flags=[]):
                 print("failed to generate image for this configuration")
 
 
+def get_inpaint_inputs():
+    os.mkdir("./test_images/inputs")
+    img_url = (
+        "https://huggingface.co/datasets/diffusers/test-arrays/resolve"
+        "/main/stable_diffusion_inpaint/input_bench_image.png"
+    )
+    mask_url = (
+        "https://huggingface.co/datasets/diffusers/test-arrays/resolve"
+        "/main/stable_diffusion_inpaint/input_bench_mask.png"
+    )
+    img = requests.get(img_url)
+    mask = requests.get(mask_url)
+    open("./test_images/input/image.png", "wb").write(img.content)
+    open("./test_images/input/mask.png", "wb").write(mask.content)
+
+
+def test_inpaint(device="vulkan", beta=False, extra_flags=[]):
```

Comment:
There is a lot of duplicated code, please add an argument that specifies inpainting and use conditionals where necessary